src/main/java/edu/tamu/app/service/manager/AbstractGitHubService.java

```diff
@@ -56,8 +60,6 @@
 
     protected final RestTemplate restTemplate;
 
-    protected GHLabel label;
```

Comment:
Should not have transient class properties on singletons that are set and used in various arbitrary functions.